usagov/vote-gov-drupal
# Vote-940 Test for government banner function and backend

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-940&assignee=611272e0eb7aef00694b8a97

## Description (optional)
This PR will add testing to the function of the government banner component for the site.  The test is checking role access along with checking that the banner is present and the drop-down function is working as expected.

## Deployment and testing (required, if applicable)

### Testing steps
1. pull down branch 
2. start local 
3. cd `testing`
4. run `npm run cy:open`
5. select `header-function.cy.js` file and verify that test is pass and working as expected
